### PR TITLE
[PUBDEV-5089] Make Booklet up-to-date with current Sparkling Water

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: h2o-docs/src/booklets/v2_2015/source/sw/sections/intro.tex [hunk 1]
```diff
@@ -84,15 +85,15 @@ \subsection{Supported Data Formats}
...
 \end{itemize}
 
 For more details, please refer to the H2O documentation at {\url{http://docs.h2o.ai}}.
 
 \subsection{Supported Spark Execution Environments}
 Sparkling Water can run on top of Spark in the following ways:
 \begin{itemize}
-  \item as a local cluster (where the master node is \texttt{local},
-\texttt{local[*]}, or \texttt{local-cluster[...]})
+  \item as a local cluster (where the master node is \texttt{local} or \texttt{local[*]})
```
Comment (jakubhava) [inline h2o-docs/src/booklets/v2_2015/source/sw/sections/intro.tex]: local-cluster is used just for testing, I think we should not expose it in the book

File: h2o-docs/src/booklets/v2_2015/source/sw/sections/intro.tex [hunk 2]
```diff
@@ -35,7 +35,7 @@ \subsubsection{Data Munging}
...
+The last use-case depicted on Figure~\ref{fig:uc3} introduces two data pipelines. The first one, called an off-line training pipeline, is invoked regularly (e.g., every hour or every day), utilizes Spark as well as H2O API and provides an H2O model as output. The H2O API allows the model to be exported in a form independent on H2O run-time. The second one processes streaming data (with help of Spark Streaming or Storm) and utilizes the model trained in the first pipeline to score the incoming data. Since the model is exported with no run-time dependency to H2O, the streaming pipeline can be lightweight and independent on H2O or Sparkling Water infrastructure.
```
Comment (jakubhava) [inline h2o-docs/src/booklets/v2_2015/source/sw/sections/intro.tex]: Code can be exported as mojo or pojo ( where mojo is not source code anymore )

**Comment by jakubhava** [comment]
@mmalohlava added documentation for backend and I'm now reading the rest of the booklet and making it up-to-date. A few more chapters to verify.

Just a minor question, may I add myself as the editor/author ( whatever makes more sense in this case, probably editor ) :) ?

**Comment by jakubhava** [comment]
TODO:
- [] Add duke library to --packages option ( not required in the next release, but that might be already late for sw booklet )

**Comment by jakubhava** [comment]
@mmalohlava @angela0xdata I think that all updates should be there. Please verify the info and possible types just to be sure before we go ahead and start printing.

Also just a minor question, may I add myself as the editor/author ( whatever makes more sense in this case, probably editor ) :) ?

**Comment by angela0xdata** [comment]
@h2o-ops please test!

**Comment by angela0xdata** [comment]
The Proofreading task is complete.

**Comment by jakubhava** [comment]
Amazing, thanks a lot @angela0xdata!

**Comment by jakubhava** [comment]
@mmalohlava do you want to have a look before it's merged or can we go ahead ?

File: h2o-docs/src/booklets/v2_2015/source/SparklingWaterBooklet.tex [hunk 3]
```diff
@@ -27,7 +27,7 @@
 \bigskip
 \textsc{\small{Michal Malohlava\hspace{20pt} Nidhi Mehta}}
```
Comment (mmalohlava) [inline h2o-docs/src/booklets/v2_2015/source/SparklingWaterBooklet.tex]: Kuba, you should be here!

File: h2o-docs/src/booklets/v2_2015/source/SparklingWaterBooklet.tex [hunk 4]
```diff
@@ -51,7 +51,7 @@
 
 Machine Learning with Sparkling Water: H2O + Spark\\
   by Michal Malohlava \&\ Nidhi Mehta\\
```
Comment (mmalohlava) [inline h2o-docs/src/booklets/v2_2015/source/SparklingWaterBooklet.tex]: + Kuba

File: h2o-docs/src/booklets/v2_2015/source/sw/sections/deployment.tex [hunk 5]
```diff
@@ -33,15 +33,15 @@ \subsubsection{Using Fatjar}
...
+\subsubsection{Using the Spark Package}
 
 Sparkling Water is also published as a Spark package. The benefit of using the package is that you can use it directly from your Spark distribution without need to download Sparkling Water.
 
 For example, if you have Spark version 2.2.0 and would like to use Sparkling Water version 2.2.2 and launch example \texttt{CraigslistJobTitlesStreamingApp}, then you can use the following command:
 
 \begin{lstlisting}[style=Bash]
 $SPARK_HOME/bin/spark-submit \
-  --packages ai.h2o:sparkling-water-core_2.11:2.2.2,ai.h2o:sparkling-water-examples_2.11:2.2.2 \
+  --packages ai.h2o:sparkling-water-core_2.11:2.2.2,ai.h2o:sparkling-water-examples_2.11:2.2.2,no.priv.garshol.duke:duke:1.2 \
```
Comment (mmalohlava) [inline h2o-docs/src/booklets/v2_2015/source/sw/sections/deployment.tex]: There should be note explaining the duke library.

**Pull request merged**